File: DayFive/Java/Task.java
import java.util.Arrays;
import java.util.Scanner;
	public class Task{
		public static void main(String[] args){
			Scanner scanner = new Scanner(System.in);
			System.out.print("Enter a valid number: ");
			String number = scanner.next();
			String result = answer(number);
			System.out.print(result);
		}
		public static String answer(String values){
			int finalResult;
			for(int counter = 0; counter < 19; counter++){
				for(int count = counter+1; count < 19; count++){
					if(counter > count){
						finalResult = counter;
						counter = count;
						count = finalResult;
				
						return "false";
					}
					else{
						return "true";
					}
				}
			}
			return values;
		}
	}	 

File: DayFive/python/task.py

finalResult = 0;
number = input("Enter a valid number: ")
if(len(number) != 16):
	print("false")
else:
	print("true")
for counter in range(19):
	for count in range(counter+1):                              
		if(counter > count):
			finalResult = counter
			counter = count
			count = finalResult

if(counter > count):
	print("False")
	
else:
	print("true")